guestbook/README.md

```diff
@@ -4,704 +4,354 @@
+Download the following configuration files:
+1. [redis-master-deployment.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-master-deployment.yaml)
+2. [redis-master-service.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-master-service.yaml)
+3. [redis-slave-deployment.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-slave-deployment.yaml)
+4. [redis-slave-service.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/redis-slave-service.yaml)
+5. [frontend-deployment.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/frontend-deployment.yaml)
+6. [frontend-service.yaml](https://kubernetes.io/docs/tutorials//docs/tutorials/stateless-application/frontend-service.yaml)
 
-**Note**:  If you are running this example on a [Google Container Engine](https://cloud.google.com/container-engine/) installation, see [this Google Container Engine guestbook walkthrough](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead. The basic concepts are the same, but the walkthrough is tailored to a Container Engine setup.
+{% endcapture %}
 
-### Prerequisites
+{% capture lessoncontent %}
 
-This example requires a running Kubernetes cluster. First, check that kubectl is properly configured by getting the cluster state:
+## Start up the Redis Master
+The guestbook application uses Redis to store its data. It writes its data to a Redis master instance and reads data from multiple Redis worker (slave) instances.
 
-```console
-$ kubectl cluster-info
-```
-
-If you see a url response, you are ready to go. If not, read the [Getting Started guides](http://kubernetes.io/docs/getting-started-guides/) for how to get started, and follow the [prerequisites](http://kubernetes.io/docs/user-guide/prereqs/) to install and configure `kubectl`. As noted above, if you have a Google Container Engine cluster set up, read [this example](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
-
-All the files referenced in this example can be downloaded [from GitHub](https://git.k8s.io/examples/guestbook).
-
-### Quick Start
+### Creating the Redis Master Deployment
+1. `cd` to the folder you saved the `.yaml.` files.
+2. Create the Redis Master Deployment from the following `.yaml` file:
 
-This section shows the simplest way to get the example work. If you want to know the details, you should skip this and read [the rest of the example](#step-one-start-up-the-redis-master).
-
-Start the guestbook with one command:
-
-```console
-$ kubectl create -f guestbook/all-in-one/guestbook-all-in-one.yaml
-service "redis-master" created
-deployment "redis-master" created
-service "redis-slave" created
-deployment "redis-slave" created
-service "frontend" created
-deployment "frontend" created
+```shell
+kubectl create -f redis-master-deployment.yaml
```

Comment:
Let's switch to using `kubectl apply`. It has more benefits over `kubectl create` ,especially when updating.